src/hotspot/share/opto/vectorization.hpp

```diff
@@ -678,15 +723,15 @@ class VPointer : public ArenaObj {
   int   invar_factor() const;
 
   // Comparable?
-  bool invar_equals(VPointer& q) {
+  bool invar_equals(const VPointer& q) const {
```

Comment:
Note: had to make some things `const` here, so that I can pass around `const VPointer&`, which I get from `_pointers.get(n)` / `get_pointer(n)`.